packages/contractkit/src/wrappers/LockedGold.test.ts

```diff
@@ -0,0 +1,43 @@
+import { newKitFromWeb3 } from '../kit'
+import { testWithGanache } from '../test-utils/ganache-test'
+import { AccountsWrapper } from './Accounts'
+import { LockedGoldWrapper } from './LockedGold'
+
+testWithGanache('Validators Wrapper', (web3) => {
+  const kit = newKitFromWeb3(web3)
+  let accounts: AccountsWrapper
+  let lockedGold: LockedGoldWrapper
+
+  // Arbitrary value.
+  const value = 120938732980
+  let account: string
+  beforeAll(async () => {
+    account = (await web3.eth.getAccounts())[0]
+    lockedGold = await kit.contracts.getLockedGold()
+    accounts = await kit.contracts.getAccounts()
+    await accounts.createAccount().sendAndWaitForReceipt({ from: account })
```

Comment:
You can set the `defaultAccount` on `kit` to avoid having to specify `from: account` on all txs